tests/playwright/page-object-model/shared/footer-navigation-component.ts

```diff
@@ -4,14 +4,20 @@ export class FooterNavigationComponent {
   readonly expect: FooterNavigationComponentAssertions
   readonly locator: Locator
   readonly privacyPolicyLinkLocator: Locator
+  readonly accessibilityStatementLinkLocator: Locator
   constructor (readonly page: Page) {
     this.expect = new FooterNavigationComponentAssertions(this)
     this.privacyPolicyLinkLocator = page.getByTestId('privacy-policy-link')
+    this.accessibilityStatementLinkLocator = page.getByTestId('accessibility-statement-link')
```

Comment:
We prefer to use `page.getByRole` rather than introducing test ids because it is closer to the user experience.
Something like `page.getByRole('link', { name: 'Accessibility statement')` should work. We can then remove the `data-testid` attribute from the actual footer code

See these links for more details - 
* https://playwright.dev/docs/locators#locate-by-role
* https://playwright.dev/docs/api/class-page#page-get-by-role